src/app/components/entityView/TranslationPopup.jsx

```diff
@@ -1,26 +1,26 @@
-import React, { PureComponent } from "react";
-import ReactMarkdown from "react-markdown";
-import i18n from "i18next";
-
-import PropTypes from "prop-types";
 import classNames from "classnames";
+import i18n from "i18next";
 import * as f from "lodash/fp";
-
-import { ColumnKinds, Langtags } from "../../constants/TableauxConstants";
+import PropTypes from "prop-types";
+import React, { PureComponent } from "react";
```

Comment:
Unused import `PureComponent`

```suggestion
import React from "react";
```